website/content/docs/internals/telemetry.mdx

```diff
@@ -49,9 +49,9 @@ The following is an example telemetry dump snippet:
 
 You'll note that log entries are prefixed with the metric type as follows:
 
-- **[C]** is a counter
-- **[G]** is a gauge
-- **[S]** is a summary
+- **[C]** is a counter. Counters are cumulative metrics that are incremented when some event occurs, and are reset at the end of reporting intervals. Vault retains counters and other metrics for one minute in-memory, so to see accurate and persistent counters over time an [aggregation solution][telemetry-stanza] must be configured.
+- **[G]** is a gauge. Gauges provide measurements of current values.
+- **[S]** is a summary. Summaries provide sample observations of values and can measure distributions of discrete events.
```

Comment:
Should we note that these types correlate to Prometheus metric types specifically https://prometheus.io/docs/concepts/metric_types/ and link to that? cause say someone familiar with Timers in statsd might not understand that Summary is used for the same thing.

Reply:
I don't think they are specific to Prometheus metric types - they might apply to Splunk too? I pulled the definition from OpenMetrics. Do you have a suggestion for how I could make this description more broadly applicable to other metrics type? I don't 100% understand, is why I ask.

Reply:
oh interesting, TIL about openMetrics 😀 ! That specification looks to be formalizing Prometheus standards, so maybe it's still specific to Prometheus? I'm not sure if there are other metric types to worry about, so just for this, I think maybe an example might make the use case clearer.
So my suggestion would be along the lines of 
```
Summaries provide sample observations of values, and is most commonly used for measuring timing durations of events.
```
but not sure if this is a kind of generalization that we'd prefer to avoid. It can be used for other things like `vault.autosnapshots.snapshot.size` also, but that is rarer.

Reply:
Ah, I see now. I've tried to summarize based on the kinds of things it seems we use them for.

Reply:
lgtm!